Index.html

```diff
@@ -38,17 +38,209 @@ <h2>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+            </div>
+
+            
+        </section>
+
+        <section class="aboutMe">
+            <h1>About me</h1>
```

Comment:
Looks like this section is having a background image :blush: 
Kindly include it. Do not forget that decorations are always background images, not `<img>` tags).